docs/cpan-dependency-confusion.md

```diff
@@ -0,0 +1,238 @@
+# CPAN dependency confusion
+## Rationale 
+Take advantage of a wrongly configured setup that resolve to public module in place of expected private/local version, because of one of the following reasons:
+- Local module is not installed at time of running CPAN client
+- Public CPAN version is preferred to local
+- Using a CPAN repository (proxy) that can be confused
+
+This type of vulnerability is presented in [Dependency Confusion](https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610) by [Alex Birsan](https://medium.com/@alex.birsan)
```

Comment:
Maybe reword this to indicate that this type of vuln has affected repositories for other languages as well, but that this can also affect CPAN

Reply:
Reworded